core-services/prow/02_config/_config.yaml

```diff
@@ -1440,7 +1441,22 @@ tide:
     - do-not-merge/hold
     - do-not-merge/work-in-progress
     - do-not-merge/invalid-owners-file
+    - do-not-merge/product-pre-release
     repos:
+    - integr8ly/installation
+    - integr8ly/integreatly-operator
+    - integr8ly/installation
+    - integr8ly/integreatly-operator
```

Comment:
This is obviously incorrect.

Reply:
That's strange. I get the `core-services/prow/02_config/_config.yaml` for this PR conflicts just now. So, I rebase it to the latest master branch. I will remove them. Thanks!

Reply:
That's strange! After I rebase the `master` branch, once run the `make update`, these configures will be added back even if I removed them manually.

Reply:
If I run the `make update` or `make prow-config`, those configures will be added back. If I don't run the `make update`, the `ci/prow/prow-config` test will fail. What should I do?

Reply:
You should do what I asked you to do in the review, so that I know what is the input to the generator.

> There are some weird changes core-services/prow/02_config/_config.yaml . I assume these come out with some interaction with the generators, but without knowing what was the original state, it is impossible to say if the input was wrong or the tool misbehaves. Would you commit your desired change in that file separately from the result or runnign the generator?

Reply:
Or to be precise - *some* stanzas moving around are expected. That's fine. The duplicate lines in the snipet above are not - these seem to be an artifact of a bad git work or something similar. When I run the generator over the **current** state of the PR, the change is fine - it just moves some bits around, which is fine.

Reply:
Now, I rebase the latest `master` branch. It looks good. Just some stanzas moving around.